.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "plugins": ["import", "require"],
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "rules": {
+    "no-console": "warn",
```

Comment:
change warn to error

Reply:
If I change to error, then we would have to delete our console logs that let us know if the server starts and if the database connects successfully.

Reply:
replace those specific console.logs with console.info

Reply:
still getting a console error:  error  Unexpected console statement -- no-console